BE/dns.py

```diff
@@ -21,7 +21,7 @@
 
 USE_FAILURE = config['dns']['use_failure_ip']
 FAILURE_IP = config['dns']['failure_ip']
-host_domain = config['dns']['domain']
+host_domain = config['dns']['domain'].strip(". ")
```

Comment:
I don't think this is needed. The user is responsible for configuring it correctly and it doesn't seem like a thing people often do and the code should handle. Though I agree there's little documentation on the format.